src/core/net/nd6.cpp

```diff
@@ -200,6 +200,95 @@ uint8_t RecursiveDnsServerOption::OptionLengthFor(uint8_t aNumAddresses)
     return ClampToUint8(DivideAndRoundUp(size, kLengthUnit));
 }
 
+//----------------------------------------------------------------------------------------------------------------------
+// Nat64PrefixInfoOption
+
+void Nat64PrefixInfoOption::Init(void)
```

Comment:
I know this class will be useful for the PERF64 implementation in `RoutingManager`, but it isn't used by any other modules yet. We generally try to avoid adding new, unused classes and prefer to add them when the code that uses them is also introduced. This keeps things consistent. What are your thoughts?